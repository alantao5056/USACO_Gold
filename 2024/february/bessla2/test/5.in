500 1000 20 10000 2
1 2 10000
2 3 10000
3 4 10000
4 5 10000
5 6 10000
6 7 10000
7 8 10000
8 9 10000
9 10 10000
10 11 10000
11 12 10000
12 13 10000
13 14 10000
14 15 10000
15 16 10000
16 17 10000
17 18 10000
18 19 10000
19 20 10000
20 21 10000
21 22 10000
22 23 10000
23 24 10000
24 25 10000
25 26 10000
26 27 10000
27 28 10000
28 29 10000
29 30 10000
30 31 10000
31 32 10000
32 33 10000
33 34 10000
34 35 10000
35 36 10000
36 37 10000
37 38 10000
38 39 10000
39 40 10000
40 41 10000
41 42 10000
42 43 10000
43 44 10000
44 45 10000
45 46 10000
46 47 10000
47 48 10000
48 49 10000
49 50 10000
50 51 10000
51 52 10000
52 53 10000
53 54 10000
54 55 10000
55 56 10000
56 57 10000
57 58 10000
58 59 10000
59 60 10000
60 61 10000
61 62 10000
62 63 10000
63 64 10000
64 65 10000
65 66 10000
66 67 10000
67 68 10000
68 69 10000
69 70 10000
70 71 10000
71 72 10000
72 73 10000
73 74 10000
74 75 10000
75 76 10000
76 77 10000
77 78 10000
78 79 10000
79 80 10000
80 81 10000
81 82 10000
82 83 10000
83 84 10000
84 85 10000
85 86 10000
86 87 10000
87 88 10000
88 89 10000
89 90 10000
90 91 10000
91 92 10000
92 93 10000
93 94 10000
94 95 10000
95 96 10000
96 97 10000
97 98 10000
98 99 10000
99 100 10000
100 101 10000
101 102 10000
102 103 10000
103 104 10000
104 105 10000
105 106 10000
106 107 10000
107 108 10000
108 109 10000
109 110 10000
110 111 10000
111 112 10000
112 113 10000
113 114 10000
114 115 10000
115 116 10000
116 117 10000
117 118 10000
118 119 10000
119 120 10000
120 121 10000
121 122 10000
122 123 10000
123 124 10000
124 125 10000
125 126 10000
126 127 10000
127 128 10000
128 129 10000
129 130 10000
130 131 10000
131 132 10000
132 133 10000
133 134 10000
134 135 10000
135 136 10000
136 137 10000
137 138 10000
138 139 10000
139 140 10000
140 141 10000
141 142 10000
142 143 10000
143 144 10000
144 145 10000
145 146 10000
146 147 10000
147 148 10000
148 149 10000
149 150 10000
150 151 10000
151 152 10000
152 153 10000
153 154 10000
154 155 10000
155 156 10000
156 157 10000
157 158 10000
158 159 10000
159 160 10000
160 161 10000
161 162 10000
162 163 10000
163 164 10000
164 165 10000
165 166 10000
166 167 10000
167 168 10000
168 169 10000
169 170 10000
170 171 10000
171 172 10000
172 173 10000
173 174 10000
174 175 10000
175 176 10000
176 177 10000
177 178 10000
178 179 10000
179 180 10000
180 181 10000
181 182 10000
182 183 10000
183 184 10000
184 185 10000
185 186 10000
186 187 10000
187 188 10000
188 189 10000
189 190 10000
190 191 10000
191 192 10000
192 193 10000
193 194 10000
194 195 10000
195 196 10000
196 197 10000
197 198 10000
198 199 10000
199 200 10000
200 201 10000
201 202 10000
202 203 10000
203 204 10000
204 205 10000
205 206 10000
206 207 10000
207 208 10000
208 209 10000
209 210 10000
210 211 10000
211 212 10000
212 213 10000
213 214 10000
214 215 10000
215 216 10000
216 217 10000
217 218 10000
218 219 10000
219 220 10000
220 221 10000
221 222 10000
222 223 10000
223 224 10000
224 225 10000
225 226 10000
226 227 10000
227 228 10000
228 229 10000
229 230 10000
230 231 10000
231 232 10000
232 233 10000
233 234 10000
234 235 10000
235 236 10000
236 237 10000
237 238 10000
238 239 10000
239 240 10000
240 241 10000
241 242 10000
242 243 10000
243 244 10000
244 245 10000
245 246 10000
246 247 10000
247 248 10000
248 249 10000
249 250 10000
250 251 10000
251 252 10000
252 253 10000
253 254 10000
254 255 10000
255 256 10000
256 257 10000
257 258 10000
258 259 10000
259 260 10000
260 261 10000
261 262 10000
262 263 10000
263 264 10000
264 265 10000
265 266 10000
266 267 10000
267 268 10000
268 269 10000
269 270 10000
270 271 10000
271 272 10000
272 273 10000
273 274 10000
274 275 10000
275 276 10000
276 277 10000
277 278 10000
278 279 10000
279 280 10000
280 281 10000
281 282 10000
282 283 10000
283 284 10000
284 285 10000
285 286 10000
286 287 10000
287 288 10000
288 289 10000
289 290 10000
290 291 10000
291 292 10000
292 293 10000
293 294 10000
294 295 10000
295 296 10000
296 297 10000
297 298 10000
298 299 10000
299 300 10000
300 301 10000
301 302 10000
302 303 10000
303 304 10000
304 305 10000
305 306 10000
306 307 10000
307 308 10000
308 309 10000
309 310 10000
310 311 10000
311 312 10000
312 313 10000
313 314 10000
314 315 10000
315 316 10000
316 317 10000
317 318 10000
318 319 10000
319 320 10000
320 321 10000
321 322 10000
322 323 10000
323 324 10000
324 325 10000
325 326 10000
326 327 10000
327 328 10000
328 329 10000
329 330 10000
330 331 10000
331 332 10000
332 333 10000
333 334 10000
334 335 10000
335 336 10000
336 337 10000
337 338 10000
338 339 10000
339 340 10000
340 341 10000
341 342 10000
342 343 10000
343 344 10000
344 345 10000
345 346 10000
346 347 10000
347 348 10000
348 349 10000
349 350 10000
350 351 10000
351 352 10000
352 353 10000
353 354 10000
354 355 10000
355 356 10000
356 357 10000
357 358 10000
358 359 10000
359 360 10000
360 361 10000
361 362 10000
362 363 10000
363 364 10000
364 365 10000
365 366 10000
366 367 10000
367 368 10000
368 369 10000
369 370 10000
370 371 10000
371 372 10000
372 373 10000
373 374 10000
374 375 10000
375 376 10000
376 377 10000
377 378 10000
378 379 10000
379 380 10000
380 381 10000
381 382 10000
382 383 10000
383 384 10000
384 385 10000
385 386 10000
386 387 10000
387 388 10000
388 389 10000
389 390 10000
390 391 10000
391 392 10000
392 393 10000
393 394 10000
394 395 10000
395 396 10000
396 397 10000
397 398 10000
398 399 10000
399 400 10000
400 401 10000
401 402 10000
402 403 10000
403 404 10000
404 405 10000
405 406 10000
406 407 10000
407 408 10000
408 409 10000
409 410 10000
410 411 10000
411 412 10000
412 413 10000
413 414 10000
414 415 10000
415 416 10000
416 417 10000
417 418 10000
418 419 10000
419 420 10000
420 421 10000
421 422 10000
422 423 10000
423 424 10000
424 425 10000
425 426 10000
426 427 10000
427 428 10000
428 429 10000
429 430 10000
430 431 10000
431 432 10000
432 433 10000
433 434 10000
434 435 10000
435 436 10000
436 437 10000
437 438 10000
438 439 10000
439 440 10000
440 441 10000
441 442 10000
442 443 10000
443 444 10000
444 445 10000
445 446 10000
446 447 10000
447 448 10000
448 449 10000
449 450 10000
450 451 10000
451 452 10000
452 453 10000
453 454 10000
454 455 10000
455 456 10000
456 457 10000
457 458 10000
458 459 10000
459 460 10000
460 461 10000
461 462 10000
462 463 10000
463 464 10000
464 465 10000
465 466 10000
466 467 10000
467 468 10000
468 469 10000
469 470 10000
470 471 10000
471 472 10000
472 473 10000
473 474 10000
474 475 10000
475 476 10000
476 477 10000
477 478 10000
478 479 10000
479 480 10000
480 481 10000
481 482 10000
482 483 10000
483 484 10000
484 485 10000
485 486 10000
486 487 10000
487 488 10000
488 489 10000
489 490 10000
490 491 10000
491 492 10000
492 493 10000
493 494 10000
494 495 10000
495 496 10000
496 497 10000
497 498 10000
498 499 10000
499 500 10000
337 143 4090
467 46 4364
120 35 3923
337 339 3371
466 379 2253
17 496 6673
461 80 3579
68 437 8265
338 215 7785
231 393 3647
7 138 5814
129 204 6473
151 370 6243
366 152 3502
99 136 1688
116 205 5698
486 94 4026
163 266 4120
3 256 92
384 376 441
145 456 9459
433 39 1981
161 4 2899
436 198 8479
362 112 9916
494 487 9716
90 249 4440
377 154 1206
133 420 6005
145 350 8925
118 140 2475
411 185 7066
251 248 9521
22 273 380
121 438 7904
168 448 6534
299 310 7784
76 232 2139
94 290 6787
214 271 3116
253 488 3224
248 22 6497
279 49 9056
130 178 6523
314 457 9588
150 229 1105
270 320 367
43 44 1915
422 90 7044
491 57 5223
138 322 5357
29 357 4386
209 102 7611
224 243 1421
409 372 794
101 477 8062
330 433 5029
355 88 5317
248 331 4689
121 123 5815
421 53 6643
19 259 299
106 144 1830
455 285 3183
368 90 7406
6 394 2246
21 96 5697
110 131 9781
161 164 3763
53 191 4025
344 116 6768
34 137 5884
257 13 1814
63 469 1045
238 167 9822
384 184 2139
242 430 2873
229 389 5861
481 172 5217
25 373 9953
119 187 9656
468 126 7604
169 166 5141
362 479 5435
67 269 8672
94 444 8927
483 61 7
308 148 1397
484 445 168
382 355 5915
313 490 4701
153 246 746
357 492 9308
223 417 3909
203 496 7365
26 428 1578
38 212 2638
362 479 5316
402 16 9076
458 304 9806
66 307 652
38 120 4980
347 242 6861
214 155 6474
342 23 2684
341 309 2705
156 241 9179
303 313 2760
315 75 2716
372 128 1218
381 30 9021
273 481 8164
344 316 5679
231 353 9937
95 190 195
360 137 3470
307 299 8437
31 483 3270
398 152 8862
54 262 2518
424 41 7435
36 366 5454
114 144 9355
256 491 4972
342 315 5044
7 423 4971
28 329 5084
1 83 3902
36 295 9249
110 103 1134
91 354 2506
232 258 9054
382 297 6459
357 210 8255
495 485 4844
326 30 925
455 14 7163
148 487 9878
255 92 2717
307 124 3585
375 416 1753
150 88 4935
367 338 7847
453 64 4899
276 264 7395
455 173 1929
432 3 3876
415 474 5868
460 462 4618
26 396 5111
100 153 3832
93 140 9092
441 283 4680
236 497 5438
105 92 5530
51 89 5476
320 141 9288
218 418 7728
376 42 2165
253 372 7917
463 456 859
476 438 7762
81 266 309
40 477 8732
187 72 9209
202 389 8822
464 261 3400
293 460 9732
52 348 5616
157 423 6348
214 494 3048
221 344 3417
288 157 2620
263 192 1556
55 179 2412
165 133 7489
324 365 4383
108 75 6218
170 92 3594
488 317 3708
460 115 9648
29 153 8159
90 49 4870
226 53 9519
419 169 1301
303 356 9861
106 329 7718
357 195 5034
295 207 5618
296 495 6484
456 378 5678
140 70 9486
8 449 3768
343 322 8292
493 239 9852
322 132 756
135 413 4139
305 351 7883
249 125 5126
376 46 3273
325 251 6725
278 351 4790
439 459 2455
228 483 1179
297 173 6654
421 448 4438
439 225 3004
64 468 9376
356 334 6981
304 131 2739
277 150 3659
270 35 6671
369 50 3344
147 469 4056
259 23 8787
447 496 2610
345 140 6599
4 58 9732
455 396 3661
312 245 1563
468 377 3393
477 484 7911
282 120 474
492 43 3683
241 496 5587
407 405 4700
149 294 7239
444 295 8330
52 1 7979
113 76 2286
381 281 551
376 12 3627
266 273 4221
75 86 6314
18 242 8094
111 64 640
32 499 3528
137 27 5938
24 333 7817
57 268 4490
478 70 2094
425 26 5211
187 36 3499
235 312 3243
403 38 5191
248 21 6608
207 435 725
240 32 2506
99 36 7221
467 405 5956
437 70 7342
217 444 5528
164 262 7432
314 268 2492
498 272 9606
265 168 8062
257 397 1688
258 290 1045
187 201 1214
39 482 9246
259 118 2299
181 69 3260
236 202 4493
301 148 8762
67 402 9437
195 373 5119
342 191 1804
494 8 1501
45 260 574
383 325 7956
218 419 7243
489 493 8453
140 17 3430
68 454 2832
424 286 336
270 119 8476
287 381 4914
135 61 5547
362 108 8693
128 224 1750
315 129 7436
171 218 746
331 164 7542
91 74 5663
495 221 6966
420 37 4263
36 190 4691
317 275 1077
129 415 8272
251 337 4641
308 204 6449
71 350 9139
25 419 9099
80 315 3671
265 372 61
415 234 6359
462 202 828
245 298 7771
228 441 3589
292 166 2178
420 428 4029
172 185 7623
443 374 8971
53 239 494
475 179 5802
72 89 6261
319 467 1909
448 369 6525
254 271 7506
301 187 7267
162 145 4479
127 225 7232
66 3 8704
163 45 7579
96 258 7374
75 248 8964
326 418 5559
462 433 3285
167 471 9136
216 381 9374
117 356 2282
197 394 3468
298 9 9296
388 51 4448
352 456 7322
245 407 3771
118 446 6173
312 123 7375
102 444 8990
60 257 7107
325 168 359
461 433 5196
237 470 638
106 227 1061
260 279 2464
315 62 4593
269 183 4571
211 328 1795
294 429 8192
275 220 5501
275 336 5808
359 310 7853
24 307 3105
77 179 5823
321 320 9778
74 197 9342
113 136 6659
131 439 8916
104 159 8362
50 214 2910
410 271 474
344 347 3604
154 39 7981
228 167 8455
34 383 2031
264 483 8292
121 411 5253
42 263 3404
133 99 3701
22 390 5758
421 406 2775
455 217 3197
211 55 5325
412 187 7020
349 254 9574
164 190 711
464 212 6214
25 474 9900
309 111 6283
291 53 4528
115 425 5260
250 431 7233
239 72 7166
446 418 5225
305 201 1729
75 326 8980
221 236 1390
150 164 5891
415 170 8882
363 408 3930
484 6 7260
468 89 6111
228 198 6851
431 270 1540
402 158 5747
216 451 8533
102 311 6615
182 274 6117
71 8 5711
94 15 1785
249 175 2101
106 411 7277
446 325 224
4 427 6181
101 199 2785
432 491 772
74 48 2565
490 451 5203
398 145 1316
335 203 6616
86 275 5758
219 287 1223
154 159 2382
195 358 6322
34 214 5328
395 114 2638
102 229 5785
290 114 2662
406 482 6348
42 216 512
482 359 1834
232 120 2393
384 84 5970
303 34 3240
32 474 1339
287 40 3864
338 473 1285
390 432 7081
388 222 326
261 298 77
302 90 9218
55 467 3146
70 44 4408
298 494 9895
97 220 8244
499 173 76
363 370 2991
329 432 7437
495 428 9202
47 56 4026
256 254 2393
234 47 820
316 204 4633
348 379 5686
263 105 6616
415 170 9246
236 330 3855
392 273 1765
81 10 6269
148 336 1440
323 166 7236
156 128 6659
232 218 9584
165 289 9572
437 316 8062
412 442 1743
367 62 6032
302 414 9180
74 338 4318
214 331 8321
496 82 2721
247 107 4937
234 66 5398
98 405 7253
411 18 128
253 272 6915
375 449 2971
71 182 2452
81 201 3071
416 495 7346
102 304 4851
92 130 6492
342 320 2937
231 200 7562
327 319 2610
133 278 5228
193 200 8249
462 136 9433
143 21 9417
261 422 2247
118 436 3382
101 58 8228
263 481 5570
172 96 64
412 52 5788
4 94 95
346 249 4369
164 304 4803
119 338 3551
386 229 7549
402 392 9450
64 449 7447
232 285 2385
352 48 8043
199 500 5227
49 267 2908
262 270 652
248 149 4963
355 256 2346
130 168 2651
452 229 7387
237 242 2973
490 277 1266
105 276 6325
200 18 555
432 349 3215
157 307 6051
21 374 8108
160 204 292
377 103 3922
295 422 2216
