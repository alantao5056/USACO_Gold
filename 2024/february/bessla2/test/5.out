14
94
95
118
140
161
164
190
258
259
271
290
427
436
486
